src/gax/src/error/rpc/mod.rs

```diff
@@ -48,6 +48,36 @@ pub struct Status {
     pub details: Vec<StatusDetails>,
```

Comment:
do we want getters for the fields? It would be a break, but more consistent with other structs in this PR.

Reply:
Punting for now.